contracts/core/tokens/rebalancing-v2/SettleRebalance.sol

```diff
@@ -118,6 +127,42 @@ contract SettleRebalance is
         return issueAmount;
     }
 
+    /**
+     * Calculates the fee and mints the rebalancing SetToken quantity to the recipient.
+     * In effect, the Set holders are paying the fee via inflation.
+     */
+    function handleFees()
+        internal
+    {
+        uint256 settlementFee = calculateRebalanceFeeInflation();
```

Comment:
Question: Do we want to enforce that the fee size is a multiple of the natural unit?